src/plus/integrations/authentication/configuredIntegrationService.ts

```diff
@@ -27,6 +29,11 @@ export type ConfiguredIntegrationType = 'cloud' | 'local';
 export class ConfiguredIntegrationService {
 	private _configured?: Map<IntegrationId, ConfiguredIntegrationDescriptor[]>;
 
+	private readonly _onDidChangeConfiguredIntegrations = new EventEmitter<void>();
+	get onDidChangeConfiguredIntegrations(): Event<void> {
+		return this._onDidChangeConfiguredIntegrations.event;
+	}
```

Comment:
This should probably suffice for listeners that care about any change, but in the near future I'll probably update this to emit the changed integration ids and how they changed (removed, added, etc.).

Reply:
Bah, this was supposed to be stashed. Removing